option.go

```diff
@@ -184,6 +196,44 @@ func FloatOption(names ...string) Option {
 func StringOption(names ...string) Option {
 	return NewOption(String, names...)
 }
+
+// StringsOption is a command option that can handle a slice of strings
 func StringsOption(names ...string) Option {
-	return NewOption(Strings, names...)
+	return DelimitedStringsOption("", names...)
+}
+
+// DelimitedStringsOption like StringsOption is a command option that can handle a slice of strings.
+// However, DelimitedStringsOption will automatically break up the associated CLI inputs based on the delimiter.
+// For example, instead of passing `command --option=val1 --option=val2` you can pass `command --option=val1,val2` or
+// even `command --option=val1,val2 --option=val3,val4`.
+//
+// A delimiter of "" means that no delimiter is used
```

Comment:
This is as opposed to strings.Split where an empty delimiter means to split on every character. We could add this behavior, but I'm not currently seeing the utility.

Reply:
@gammazero @lidel I pushed an update here that panics when an empty delimiter is passed, but still uses the same underlying struct for StringsOption and DelimitedStringsOption with StringsOption using an empty delimiter. let me know what you think and we'll merge 🚢